diff --git a/.venv/lib/python3.11/site-packages/sounddevice.py b/.venv/lib/python3.11/site-packages/sounddevice.py
index a94f43b..4c82281 100644
--- a/.venv/lib/python3.11/site-packages/sounddevice.py
+++ b/.venv/lib/python3.11/site-packages/sounddevice.py
@@ -59,16 +59,19 @@ from _sounddevice import ffi as _ffi


 try:
-    for _libname in (
-            'portaudio',  # Default name on POSIX systems
-            'bin\\libportaudio-2.dll',  # DLL from conda-forge
-            'lib/libportaudio.dylib',  # dylib from anaconda
-            ):
-        _libname = _find_library(_libname)
-        if _libname is not None:
-            break
+    if _os.environ.get('PORTAUDIO_LIBNAME', ''):
+        _libname = _lib = _os.environ.get('PORTAUDIO_LIBNAME')
     else:
-        raise OSError('PortAudio library not found')
+        for _libname in (
+                'portaudio',  # Default name on POSIX systems
+                'bin\\libportaudio-2.dll',  # DLL from conda-forge
+                'lib/libportaudio.dylib',  # dylib from anaconda
+                ):
+            _libname = _find_library(_libname)
+            if _libname is not None:
+                break
+        else:
+            raise OSError('PortAudio library not found')
     _lib = _ffi.dlopen(_libname)
 except OSError:
     if _platform.system() == 'Darwin':
